test/test_torch.py

```diff
@@ -14581,10 +14578,6 @@ def test_helper(x, memory_format):
             self.assertFalse(like.is_contiguous())
             self.assertTrue(like.is_contiguous(memory_format=memory_format))
 
-            sparse = x.to_sparse()
```

Comment:
This (sub)test can be added back, right?

Reply:
yes, you're right. I somehow missed that. I'll get it back. Thanks !